recipes-nodes/rbuilder-status-watchdog/rbuilder-status-watchdog.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/sh
+### BEGIN INIT INFO
+# Provides:          rbuilder-status-watchdog
+# Required-Start:    $network
+# Required-Stop:
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: rbuilder status watchdog
+# Description:       Checks the status of rbuilder in BuilderHub and toggles the service
+### END INIT INFO
+
+set -e
+
+SYSTEM_API_FIFO="/var/volatile/system-api.fifo"
+RBUILDER_PERSISTENT_DIR="/persistent/rbuilder"
+RBUILDER_STATUS_FLAG_FILE="${RBUILDER_PERSISTENT_DIR}/rbuilder.enabled"
+BHUB_URL="http://localhost:7937/api/l1-builder/v1/configuration"
+
+log() {
```

Comment:
eventually, we should put this log function into a `tools` shell script that gets sources by other scripts, instead of duplicating it in every one? :)